haskell-api.md

```diff
@@ -75,20 +89,6 @@ But here, it would be `schools.id`: `/3/students?schools.id=1,2,3`.
 If a filter exists that is not for an attribute present in the response, the
 name can be inferred by what it would look like if it were.
 
-### Semantics
-
-Prefer `IN` equality semantics.
-
-- Good: `{attribute}={value1},{value2},{value3},...`
-- Bad: `{attribute}={value}`
-
-This is because:
```

Comment:
I didn't think this justification was super valuable, so once I mentioned the "behave like `[in]`" in the new section, I figured we could drop all this. If anyon disagrees, I can move this content up there instead.